creation/views.py

```diff
@@ -69,19 +69,19 @@ def is_external_contributor(user):
 
 def is_videoreviewer(user):
     """Check if the user is having video reviewer rights"""
-    if user.groups.filter(name='Video-Reviewer').count() == 1:
+    if user.groups.filter(name='Video-Reviewer').count() :
```

Comment:
This can be updated to:

```
def is_videoreviewer(user):
  return user.groups.filter(name='Video-Reviewer').exists()
```

This is much more performant than using `count()`.